src/lando/utils/management/commands/setup_dev.py

```diff
@@ -42,10 +44,12 @@ def setup_workers(self):
         for repo in Repo.objects.all():
             # Associate all repos with applicable worker.
             for worker in workers.values():
-                if not worker or worker.scm != repo.scm_type:
+                if worker.scm != repo.scm_type:
                     continue
-                self.stdout.write(f"Adding {repo} ({repo.scm_type}) to {worker}.")
-                worker.applicable_repos.add(repo)
+                # if not worker.applicable_repos.filter(repo=repo):
+                if repo.worker_set.exists():
```

Comment:
nit: we should probably put a nicer `related_name` in that model, so we can do something like `repo.workers`.

Reply:
I'd rather not fiddle with this here, but I agree.